hs_model_program/forms.py

```diff
@@ -76,11 +76,10 @@ def __init__(self, allow_edit=True, res_short_id=None, element_id=None, *args, *
         super(mp_form, self).__init__(*args, **kwargs)
         self.helper = mp_form_helper(allow_edit, res_short_id, element_id, element_name='MpMetadata', files=files)
 
+        # hide the field help text
         for field in self.fields:
```

Comment:
Do you see the help text on the web page if you don't hide it using the for loop code? If Yes then leave it as it is.